esp/src/eclwatch/stub.js

```diff
@@ -20,9 +21,11 @@ define([
     "css!hpcc/css/hpcc.css"
 
 ], function (fx, dom, domStyle, ioQuery, ready, lang, arrayUtil, topic,
-    Utility, Session, LockDialogWidget,
+    Utility, Session, KeyValStore, LockDialogWidget,
     entities, Toaster) {
 
+    const store = KeyValStore.globalKeyValStore()
+    store.set("", "", false, "HPCCApps", "ECLWatch");
```

Comment:
@rpastrana is this enough to work around the missing section issue?

Reply:
@GordonSmith yes this is enough to create the NS. We currently don't have a createNS() method, but could add to make this process cleaner.

Reply:
Ideally you would just create it as needed?